roles/openshift_on_openstack/tasks/main.yml

```diff
@@ -158,21 +158,36 @@
     dest: "{{ all_yml_path }}"
     backup: yes
 
-# Add a block of configuration to the end of the OSEv3 file.
-- name: Adding a block of configuration to the end of inventory/group_vars/OSEv3.yml
-  blockinfile:
+- name: Create the OSEv3.yml.cfg (OSEv3.yml override) file.
+  template:
+    dest: "{{ osev3_yml_cfg_path }}"
+    src: OSEv3.yml.cfg.j2
+
+- name: Reading the contents of {{ osev3_yml_path }}
+  command: "cat {{ osev3_yml_path }}"
+  register: l_osev3_yml
+  changed_when: false
+
+- name: Reading the contents of {{ osev3_yml_cfg_path }}
+  command: "cat {{ osev3_yml_cfg_path }}"
+  register: l_osev3_yml_cfg
+  changed_when: false
+
+- name: Setting facts from {{ osev3_yml_path }} and 'templates/OSEv3.yml.cfg.j2'
```

Comment:
Use the variable so we can track down the file: "Setting facts from {{ osev3_yml_path }} and {{ osev3_yml_cfg_path}}"